slash/general.py

```diff
@@ -32,6 +32,57 @@ async def github(self, interaction: Interaction):
             embed=embed, view=Link("https://github.com/codeman1o1/TIJK-Bot")
         )
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def avatar(
+        self,
+        interaction: Interaction,
+        user=SlashOption(
+            description="The user of which you wish to download their pfp",
+            required=True,
+        ),
+    ):
+        """Get the avatar of you or someone else"""
+        user = user or interaction.user
+
+        avatars_list = []
+
+        def target_avatar_formats(avatar):
+            formats = ["JPEG", "PNG", "WebP"]
+            if avatar.is_animated():
+                formats.append("GIF")
+            return formats
+
+        if not user.avatar and not user.guild_avatar:
+            return await interaction.send(f"**{user}** has no avatar set, at all...")
+
+        if user.avatar:
+            avatars_list.append(
+                "**Account avatar:** "
+                + " **-** ".join(
+                    f"[{img_format}]({user.avatar.replace(format=img_format.lower(), size=1024)})"
+                    for img_format in target_avatar_formats(user.avatar)
+                )
+            )
+
+        embed = nextcord.Embed(colour=user.top_role.colour.value)
```

Comment:
Please use `color` instead of `colour`. While this is not wrong, `color` is used everywhere else.
```suggestion
        embed = nextcord.Embed(color=user.top_role.color.value)
```

Reply:
I will let you change it for TIJK-Bot, because it is your bot, but i prefer British English, that's why i did it like this.